Automattic/woocommerce-payments
# Add loading icon to UPE

Fixes #1987 

#### Changes proposed in this Pull Request
This PR adds a loading icon to the UPE section of the payments area during the classic checkout. This was needed as the UPE flow requires a little more time to load than the rest of the checkout areas and in the meantime the section appeared broken. Due to the wide variablity of how the UPE may display, a simple loading icon was used here to match the existing look and feel of the checkout loading areas.

Animation of this implementation: https://d.pr/i/F2jdhj

#### Testing instructions

* Start with this PR pulled down (and assets recompiled).
* Enable UPE (_wcpay_feature_upe set to 1 in wp_options)
* Add item to cart and go to checkout
* When checkout page is loading view the payments area to confirm that a loading icon is displayed and then replaced by the UPE.
* If you have any saved payment methods, this behavior will be hidden under the Use a new payment method radio button. So it is best to test with no payment methods saved.

-------------------
